src/Doc/CMakeLists.txt

```diff
@@ -21,14 +21,14 @@ if(BUILD_GUI)
     configure_file(freecad.qhc ${CMAKE_BINARY_DIR}/doc/freecad.qhc COPYONLY)
     INSTALL(FILES
         ${CMAKE_BINARY_DIR}/doc/freecad.qch
-        DESTINATION ${CMAKE_INSTALL_DOCDIR}
+        freecad.qhc

```

Comment:
As mentioned in the tracker, `freecad.qhc` and `freecad.qch` are compiled HTML documentation. Which is the appropriate place for this? Data or documentation folders?

Reply:
As I mentioned on the tracker, since it's compiled and specific to Qt, it doesn't look like something people are going to looking for in /usr/share/doc., so DATADIR would be better.